contrib/pkg/broker/server/server.go

```diff
@@ -146,19 +164,22 @@ func (s *server) bind(w http.ResponseWriter, r *http.Request) {
 	if result, err := s.controller.Bind(instanceID, bindingID, &req); err == nil {
 		util.WriteResponse(w, http.StatusOK, result)
 	} else {
-		util.WriteResponse(w, http.StatusBadRequest, err)
+		util.WriteErrorResponse(w, http.StatusBadRequest, err)
 	}
 }
 
 func (s *server) unBind(w http.ResponseWriter, r *http.Request) {
 	instanceID := mux.Vars(r)["instance_id"]
 	bindingID := mux.Vars(r)["binding_id"]
+	q := r.URL.Query()
+	serviceID := q.Get("service_id")
+	planID := q.Get("plan_id")
```

Comment:
looked at the spec https://github.com/openservicebrokerapi/servicebroker/blob/master/spec.md#parameters-3 I get it.